### ISSUE TITLE
Documentation not working for short-assignment inferred variables

### ISSUE THREAD

**Issue by gauravvnera** [go-lang-plugin-org/go-lang-idea-plugin#1682]
Consider the below snippet of code:
```go
client := &http.Client{}
resp, err := client.Get("http://gobootcamp.com")
fmt.Println(resp.Body, err)
```
The popup for documentation gets stuck at "Fetching Documentation..." for 
1) client
2) resp
3) resp.Body
etc.

plugin: 0.9.348

**Comment by dlsniper** [comment]
Hi, what would be the expected result there? There is no docblock for any of the variables you've mentioned. Maybe we could infer the type?

**Comment by gauravvnera** [comment]
If we could infer the type, then 
1) client would show type of the variable
2) resp would show type of the variable
3) resp.Body would show doc from http.Response.Body

**Comment by dlsniper** [comment]
@zolotov and @ignatov I'll leave this open as it's an extension of #1676.
As for type inference, until we can do that, should the plugin say something like: "No documentation found" rather that the "Fetching Documentation..." message? If so, I'll work on a patch for it today.

**Comment by gauravvnera** [comment]
thanks for taking the request!

**Comment by dlsniper** [comment]
@gauravvnera thanks for reporting ;)

**Comment by zolotov** [comment]
3) Resp.Body will be fixed automatically after #1676
